.github/actions/nix-run/action.yml

```diff
@@ -1,12 +1,11 @@
 name: Runs a command within our nix-shell
-description: Runs a command within our nix-shell with properly set-up substitutoters and binary caches
+description: Runs a command within our nix environment
 inputs:
   command:
     required: true
     description: The command to run
 runs:
   using: "composite"
   steps:
-  - run: |
-          nix-shell --pure --run "${{ inputs.command }}" --allowed-uris 'https://github.com/NixOS/nixpkgs https://github.com/input-output-hk https://github.com/NixOS/nixpkgs-channels' --option trusted-public-keys "hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" --option substituters "https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/" nix/ci.nix
+  - run: nix develop .#ci --command "${{ inputs.command }}"
```

Comment:
Shouldn't it be the same without quotes? Will that not fail on multiple-word-long commands?

Reply:
It seems to work just fine.